src/appengine/handlers/commit_range.py

```diff
@@ -15,11 +15,12 @@
 
 import json
 
+from flask import request
```

Comment:
Forgot to mention this, but there should be a newline in between third party dependencies and our own libraries.

Reply:
Sure, will fix this as part of next PR.